src/Elastic.Documentation.Site/Assets/open-details-with-anchor.ts

```diff
@@ -1,23 +1,66 @@
-import { UAParser } from 'ua-parser-js'
-
-const { getBrowser } = new UAParser()
-
-// This is a fix for anchors in details elements in non-Chrome browsers.
+// Opens details elements (dropdowns) when navigating to an anchor link within them
+// This enables deeplinking to collapsed dropdown content
 export function openDetailsWithAnchor() {
     if (window.location.hash) {
         const target = document.querySelector(window.location.hash)
         if (target) {
             const closestDetails = target.closest('details')
             if (closestDetails) {
-                const browser = getBrowser()
-                if (browser.name !== 'Chrome') {
-                    closestDetails.open = true
+                closestDetails.open = true
+                // Small delay to ensure the details element is open before scrolling
+                setTimeout(() => {
```

Comment:
I think this was conditional because chrome already ensures the parent content is visible. I think `instant` might be more accessabe than 'smooth'. Can you confirm?